src/dlgTriggerEditor.h

```diff
@@ -416,7 +416,7 @@ private slots:
     void autoSave();
     void setupPatternControls(const int type, dlgTriggerPatternEdit* pItem);
     void key_grab_callback(const Qt::Key, const Qt::KeyboardModifiers);
-
+    void setShortcuts(const bool unsetInstead = false);
```

Comment:
Apropos a previous comment - and given that (unlike electronic circuit diagrams where an overline or failing that an underline prefix/suffix like: `_action_` indicate a inverted logic connection) we cannot easily indicate a negative state/action, how about:
```suggestion
    void setShortcuts(const bool setNotUnset = true);
```
which says what it will do if `true` "set" (and/or `false` "NotUnset")?

Reply:
Fair enough